### PR TITLE
All the berry info

### PR DESCRIPTION
**Pull request by Bond2322**
this is honest truth. Isn't is crazy!

### CODE REVIEW

File: abc.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ New start line by Bond!
 This is an OLD file!
 This is a new line pushed here!
 THis is a new line by Bond!
+Strawberry is not a berry.
```
Comment (Imrul2322) [inline abc.txt:5]: This is awesome. Nice work Bond!

**Review by Imrul2322 — CHANGES_REQUESTED** [review]

File: abc.txt (hunk 1 above)
Comment (Bond2322) [reply abc.txt:5]: I changed it accordingly. Thanks.

**Review by Bond2322 — COMMENTED** [review]
Changed accordingly.

**Review by Bond2322 — COMMENTED** [review]
updated!

**Pull request merged**